lib/internal/errors.js

```diff
@@ -118,6 +118,7 @@ E('ERR_HTTP_HEADERS_SENT',
 E('ERR_HTTP_INVALID_STATUS_CODE', 'Invalid status code: %s');
 E('ERR_HTTP_TRAILER_INVALID',
   'Trailers are invalid with this transfer encoding');
+E('ERR_ICU_NOT_SUPPORTED', '%s is not supported on Node.js built without ICU');
```

Comment:
Could you add a test asserting the expected message, to `test/parallel/test-internal-errors.js`, as per the new tweaks in https://github.com/nodejs/node/blob/master/doc/guides/using-internal-errors.md#testing-new-errors